x-pack/plugins/cases/public/components/configure_cases/index.tsx

```diff
@@ -94,6 +94,20 @@ export const ConfigureCases: React.FC = React.memo(() => {
     [refetchActionTypes, refetchCaseConfigure, refetchConnectors, setEditedConnectorItem]
   );
 
+  const onConnectorCreated = useCallback(
+    async (createdConnector) => {
+      const caseConnector = normalizeActionConnector(createdConnector);
+
+      onConnectorUpdated(createdConnector);
+      setConnector(caseConnector);
+      persistCaseConfigure({
+        connector: caseConnector,
+        closureType,
+      });
```

Comment:
I think is best if we wait for the new connector to be persisted in the configuration and then update the connector.

```
await persistCaseConfigure({
    connector: caseConnector,
    closureType,
});

onConnectorUpdated(createdConnector);
setConnector(caseConnector);
```